index.js

```diff
@@ -1,155 +0,0 @@
-const readline = require('readline');
-const rl = readline.createInterface(process.stdin, process.stdout);
-
-function ask(questionText) {
-  return new Promise((resolve, reject) => {
-    rl.question(questionText, resolve);
-  });
-}
-//the point of this function is to choose which game to play
-games (); 
-async function games() {
-  let chooseGame = "d"; 
-  //Choose a game and the appropriate game will run 
-
-  while ((chooseGame.toLowerCase().trim() !== "a") && (chooseGame.toLowerCase().trim() !== "b") && (chooseGame.toLowerCase().trim() !== "c")) {
-  chooseGame = await ask(`Welcome! Please see the following menu and pick your game by entering the letter of your choice: 
-  a.) Play Computer Mind Reader - let the computer guess your secret number!
-  b.) Play Player Guesser Game - try to guess the computer's game!
-  c.) These games are lame and I am offended you think I would find them entertaining. Goodbye.\n`) 
-  if (chooseGame.toLowerCase().trim() == "a") {
-    start();
-  } else if (chooseGame.toLowerCase().trim() == "b") {
-    inverse(); 
-  } else if (chooseGame.toLowerCase().trim() == "c") {
-      process.exit() 
-    }
-  }
-  }
-
-async function start() { 
-
-  //opening message 
-  await ask(`\nWelcome to Computer Mind Reader! You will think of a number and I will use my superior clairvoyance to guess your secret number.
-  \nPress any key when you've thought of your number.\n`);
-
-  //starting values for max and min, declare min, max, guess variable 
-  let max = await ask("Give me a number that is higher than your number - this will be the maximum number that I will guess.\n");
-  max = +max;
-  let min = 1;
-  let guess = 0
-
-  //set up a way to keep track of guesses so can caluclate number of guesses 
-  let numbersGuessed = [];
-  let count = 0; 
-  function counter () {
-    count = numbersGuessed.length
-    return count 
-  }; 
```

Comment:
Sections like this a great for breaking down what you're using this for. But to save lines of code you could remove lines 44-48, and when you wanted to console.log out what the number of guesses were, you could just do a `console.log(`It took ${numberGuessed.length()} tries to guess your number)